dev_tools/oneapi/convert_source.sh

```diff
@@ -364,7 +364,7 @@ if [ "${EXTRACT_KERNEL}" = "true" ]; then
         dpct_device_path=$(echo "${variable}" | sed 's/common/dpcpp_code/g')
         dpct_device_file=$(echo "${dpct_device_path}" | sed 's|/|@|g')
         dpct_device_file="output/${dpct_device_file}"
-        cat ${dpct_file} | sed -n "/${device_regex} - start/,/${device_regex} - end/p" | sed "1d;\$d" > ${dpct_device_file}
+        cat "${dpct_file}" | sed -n "/${device_regex} - start/,/${device_regex} - end/p" | sed "1d;\$d" > ${dpct_device_file}
```

Comment:
Minor, was this one creating a warning
```suggestion
        cat "${dpct_file}" | sed -n "/${device_regex} - start/,/${device_regex} - end/p" | sed "1d;\$d" > "${dpct_device_file}"
```

Reply:
Yes, I overlooked it. Generally `cat "{dpct_file}" | cmd` is also a bit useless, which overshadowed the globbing catch a bit.

Alternative would be `cmd < ${dpct_file}`, but I didn't think that would be critical at all.